wiki/osu!dev_Discord_server/de.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+  - development
+  - Softwareentwicklung
+---
+
+# osu!dev Discord-Server
+
+**osu!dev** ist der offizielle [Discord](https://discordapp.com)-Server, um an den Open-Source und Community-Projekten von osu! zu arbeiten. Das ist der Marktplatz, um mit dem [osu!-Team](/wiki/People/The_Team) in Kontakt zu treten und an der Entwicklung teilzunehmen.
```

Comment:
```suggestion
**osu!dev** ist der offizielle [Discord](https://discordapp.com)-Server, um an den Open-Source und Community-Projekten von osu! zu arbeiten. Das ist das Forum, um mit dem [osu!-Team](/wiki/People/The_Team) in Kontakt zu treten und an der Entwicklung teilzunehmen.
```

Just a suggestion, but I think this reads better. With `Marktplatz` I really think of trading more than anything else.